datameta/email/smtp.py

```diff
@@ -77,7 +77,7 @@ def sendMessage(self,
             part = MIMEBase("application", app_type)
             part.set_payload(data)
             email.encoders.encode_base64(part)
-            bfname = fname.encode('utf-8')
+            # bfname = fname.encode('utf-8')
```

Comment:
can go?

Reply:
I wanted to keep it in there as a reminder, haven't figured out yet if the current code may be broken 😅